src/database/migrations/1.0.0/1696782681632-SystemServicesNode.ts

```diff
@@ -0,0 +1,85 @@
+import {MigrationInterface, QueryRunner, Table} from "typeorm";
+
+export class SystemServicesNode1696782681632 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'SYS', NULL, 'System')"
+        );
+
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'SYS', NULL, 'DHCP')"
+        );
+
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'SYS', NULL, 'Keepalived')"
+        );
+
+        await queryRunner.query(
+            "INSERT INTO `fwc_tree_node_types` (`node_type`, `obj_type`, `name`) VALUES( 'SYS', NULL, 'HAProxy')"
+        );
+
+        const firewalls = await queryRunner.query(
+            "SELECT `id` FROM `fwc_tree` WHERE `node_type` = 'firewall'"
+        );
+
+        for (const firewall of firewalls) {
+            await queryRunner.query(
+                "INSERT INTO `fwc_tree` (`parent_id`, `name`, `node_type`) VALUES (?, 'System', 'SYS')",
+                [firewall.id]
+            );
+        }
```

Comment:
This is wrong, `node_type` will never have the value `firewall`.
Look at the contents of the `fwc_tree_node_types` table:
```mysql
mysql> select * from fwc_tree_node_types;
+-----------+----------+---------------------------+---------------+------------+
| node_type | obj_type | name                      | api_call_base | order_mode |
+-----------+----------+---------------------------+---------------+------------+
| CA        |     NULL | CA                        | NULL          |          2 |
| CL        |     NULL | Cluster                   | NULL          |          1 |
| COD       |       24 | Country Objects           | NULL          |          1 |
| COF       |       22 | Folder Countries          | NULL          |          1 |
| CON       |       23 | Continent Objects Folder  | NULL          |          1 |
| CRT       |     NULL | Certificate               | NULL          |          2 |
| FCA       |     NULL | Folder CA                 | NULL          |          2 |
| FCF       |     NULL | Folder Cluster Firewalls  | NULL          |          2 |
| FCR       |     NULL | Folder CRT                | NULL          |          2 |
| FD        |     NULL | Folder                    | NULL          |          1 |
| FDC       |     NULL | Folder Clusters           | NULL          |          2 |
| FDF       |     NULL | Folder Firewalls          | NULL          |          2 |
| FDI       |       10 | Folder Interfaces         | NULL          |          2 |
| FDO       |     NULL | Folder Objects            | NULL          |          1 |
| FDS       |     NULL | Folder Services           | NULL          |          1 |
| FDT       |     NULL | Folder Times              | NULL          |          1 |
| FP        |     NULL | FILTER POLICIES           | NULL          |          1 |
| FP6       |     NULL | FILTER POLICIES IPv6      | NULL          |          1 |
| FW        |     NULL | Firewall                  | NULL          |          1 |
```

The `node_type` for a firewall will have the value `FW`.

You should name the constant `firewall_nodes` instead of `firewalls` because you are getting the tree nodes that are associated with a firewall, not the firewalls temselves.

And what about the clusters ?